tensorflow-ops/tests/GradientTest.hs

```diff
@@ -392,18 +392,18 @@ transAttrs a b =
 testConv2DBackpropInputGrad :: Test
 testConv2DBackpropInputGrad = testCase "testConv2DBackpropInputGrad" $ do
     (dx, shapeDX, shapeX) <- TF.runSession $ do
-        let conv_input_shape = TF.vector [1, 2, 2, 1 :: Int32] -- [batch, h, w, in_channels]
```

Comment:
Looks like an unrelated cleanup. Split it off?

Reply:
Reverted.